set-pr-milestone.py

```diff
@@ -16,8 +16,9 @@
 patch_url = f"https://api.github.com/repos/{owner}/{repo}/issues/{args.pr}"
 
 headers = {
-    "Accept": "application/vnd.github.v3+json",
-    "Authorization": f"token {os.environ['GH_TOKEN']}",
+    "Accept": "application/vnd.github+json",
+    "Authorization": f"Bearer {os.environ['GH_TOKEN']}",
+    "X-GitHub-Api-Version": "2022-11-28"
```

Comment:
where is this coming from, I mean, would this date stay the same?

Reply:
That's the recommended way to version the API from their docs.